ContactDetailsApi/V2/Factories/EntityFactory.cs

```diff
@@ -14,10 +14,13 @@ public static class EntityFactory
     {
         public static ContactDetails ToDomain(this ContactDetailsEntity databaseEntity)
         {
+            if (databaseEntity == null) return null;
+            if (databaseEntity.ContactInformation == null) return null;
```

Comment:
Are we sure about returning `null` root object entity when some of its nested data is `null`?

If it's supposed to be that way, perhaps, it would be useful to leave a code comment about this? (As this is an unusual pattern that can easily look like a mistake)